.github/workflows/exhaustive_package_test.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           cat all_packages_deps_errors_* > all_deps_errors_lf.txt
           tr -d '\r' < all_deps_errors_lf.txt > reports/all_deps_errors.txt
       - name: Store collated and raw reports as artifacts
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: reports-final
```

Comment:
@dukecat0 this **will break** if you actually trigger the workflow.

See https://github.com/orgs/aio-libs/discussions/31 / https://github.com/actions/upload-artifact/issues/472.

Reply:
Thanks for the heads up! I will revert it first.